pandas/core/arrays/integer.py

```diff
@@ -354,6 +354,13 @@ def __init__(self, values: np.ndarray, mask: np.ndarray, copy: bool = False):
             )
         super().__init__(values, mask, copy=copy)
 
+    def __iter__(self):
```

Comment:
I think we could actually move this to base masked and/or the base EA interface (maybe)